### ISSUE TITLE
Replicating analysis on Xenium lung cancer dataset

### ISSUE THREAD

**Issue by caimaco** [dcjones/proseg#142]
Hi,

I am trying to replicate the results from the [Proseg paper](https://www.nature.com/articles/s41592-025-02697-0#Sec9), specifically with the [Xenium lung cancer dataset](https://www.10xgenomics.com/datasets/preview-data-ffpe-human-lung-cancer-with-xenium-multimodal-cell-segmentation-1-standard). I am not able to replicate the plots in Figure 5, and would like to ask what parameters you used to run Proseg, and also in the preprocessing and subsequent analysis.

I used Proseg v3.1.0 with the following command:
```
proseg --output-spatialdata output_replication.zarr --output-expected-counts replication_expected_counts.mtx.gz --xenium ../transcripts.parquet
```

For the analysis, I used the following code to filter out cells with less than 10 transcripts and take the censored log proportions, as per the paper.
```
sc.pp.filter_cells(adata, min_counts=10)

cell_sums = adata.X.sum(axis=1)
temp_X = adata.X.copy()
temp_X = temp_X.toarray().astype(np.float64)
temp_X = temp_X / cell_sums
temp_X = np.clip(temp_X, a_min = 1e-4, a_max = None)
temp_X = np.log(temp_X)

adata.X = temp_X.copy()
```

Then I used the following code to perform dimensionality reduction and clustering.
```
sc.pp.neighbors(adata, n_neighbors = 15, use_rep = 'X')
sc.tl.umap(adata)
sc.tl.leiden(adata, resolution= 1)
sc.pl.umap(adata, color='leiden')
```

The UMAPs I got did not resemble those of Figure 5A - I have attached here the plots. If you could let me know how you analysed the Xenium lung cancer sample, I would greatly appreciate it. Thank you very much! 

<img width="581" height="429" alt="Image" src="https://github.com/user-attachments/assets/48da2977-6634-4cf5-9b6f-20580fa56af3" />

<img width="581" height="429" alt="Image" src="https://github.com/user-attachments/assets/cdf6b358-d204-478c-af97-02d1d820f326" />

**Comment by dcjones** [comment]
I believe the figure in the paper was done a version close to v1.0.0. A great deal has changed since then, so I wouldn't be surprised if it looks somewhat different now.

The transformation you are doing on count is what I did in the paper, so that's not the issue. The only other difference I can think of, is that I ran umap directly rather than through scanpy, but I'm not sure why that would matter. Here's what I did:
```python
nneighbors = 15
knn = umap.umap_.nearest_neighbors(
        X,
        n_neighbors=nneighbors,
        metric="euclidean",
        metric_kwds=None,
        angular=False,
        random_state=None)

Xumap = umap.UMAP(n_neighbors=nneighbors, precomputed_knn=knn).fit_transform(X)
adata.obsm["Xumap"] = Xumap
```
Then I reused the same knn graph for the leiden clustering:
```python
edges = []
for i in range(knn[0].shape[0]):
    edges.extend([(i, j) for j in knn[0][i,:]])

G = igraph.Graph(adata.shape[0], edges)
partition = leidenalg.find_partition(
    G, leidenalg.RBConfigurationVertexPartition, resolution_parameter=2.0)
adata.obs["leiden"] = partition.membership
```

**Comment by caimaco** [comment]
Hi,

Thanks so much for your response! Would it be possible to obtain the data that was used in the paper, in particular the cell-type labels for the [Xenium lung cancer dataset](https://www.10xgenomics.com/datasets/preview-data-ffpe-human-lung-cancer-with-xenium-multimodal-cell-segmentation-1-standard) used in Figure 5, in order to replicate the figure/s? 

Thank you very much!
